defaults/main.yml

```diff
@@ -1456,6 +1456,26 @@ owncloud__nginx_access_log_assets: True
 # TODO: Confirm that this variable does what it says.
 owncloud__nginx_client_body_temp_path: ''
 
+owncloud__nginx__php_status: False
+owncloud__nginx__php_status_location:
+  pattern: '~ ^/(php_status|php_ping)$'
+  options: |
+    access_log off;
+    {% if nginx_status_localhost %}
+    {% for address in nginx_status_localhost | unique %}
+    allow {{ address }};
+    {% endfor %}
+    {% endif %}
+    {% if nginx_status %}
+    {% for address in nginx_status | unique %}
+     allow {{ address }};
+    {% endfor %}
+    {% endif %}
+    deny all;
+    include fastcgi_params;
+    fastcgi_index index.php;
+    fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+    fastcgi_pass php_owncloud;
```

Comment:
This should be moved to `debops.nginx` because the context from which `debops.owncloud` is running might not have `nginx_status_localhost` defined which would result in idempotent problems.